src/main/java/pl/shop/controllers/RestController.java

```diff
@@ -0,0 +1,56 @@
+package pl.shop.controllers;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import pl.shop.service.Service;
+
+/**
+ * Komencie wrzucam wyłącznie tymczasowe. Przypominam, że zgodnie z przykazanimi Wujka Boba,
+ * kod ma się komentować sam, więc to tylko tak na start. :)
+ *
+ * Kontrolery dwa ze względu na rozdział na samo API oraz wersję dla użytkownika. RestController
+ * to nasze API -> będzie można sobie testować np. w Postmanie dla fejmu.
+ *
+ * Zwracam uwagę na adnotację RequestMapping nad klasą. Oznacza to tyle co String w nawiasie mówi.
+ * To jest API. W celu dostania się pod którykolwiek z endpointów, należy dodać tegoż właśnie Stringa
+ * do URLa.
+ * Wydaje mi się, że w warunkach bitewnych back-endowcy robiliby samo właśnie API (a więc RestController)
+ * i na podstawie tego frontowcy robiliby resztę.
+ * My natomiast frontowców chyba nie mamy, więc radzimy sobie sami i z tego powodu ThymeleafController. :)
+ */
+
+@org.springframework.web.bind.annotation.RestController
+@RequestMapping("/api/v1")
+public class RestController {
+
+    Service service;
+
+    public RestController(Service service) {
+        this.service = service;
+    }
+
```

Comment:
Service można wstrzyknąć przez konstuktor używając adnotacji @RequiredArgsConstructor (lombok).

Reply:
To ciekawe, nie znałem tego. Zmienimy przy następnym feature'rze, dziękować. :)